website/docs/reference/dbt-classes.md

```diff
@@ -167,3 +167,201 @@ The execution of a resource in dbt generates a Result object. This object contai
     'required': ['node'],
 }
 ```
+
+
+## SQL Results
+:::warning These docs are a work in progress
+:::
+
+Note for the discerning reader: this is not a true python Class — it's a dictionary.
+
+The [`load_result` macro](load_result) macro returns a SQL Results object. The schema of a SQL Results Object is as follows ([source code](https://github.com/fishtown-analytics/dbt/blob/dev/marian-anderson/core/dbt/context/providers.py#L667)):
+
+```python
+{
+    'status': 'SUCCESS 1',
+    'data': [(Decimal('1'),)],
+    'table': <agate.table.Table object at 0x11466ad90>
+}
+```
+
+Note that this contract may be subject to change in the future until dbt's API has reached a "stable" state.
+
+Keys:
+- `status`: The status code returned by the database. Note: this varies across databases and query types.
+- `data`: An array of tuples represent the results. The first item represents the first row. For example, for a table with results:
+
+| my_col_1 | my_col_2 |
+|----------|----------|
+| 1        | 2        |
+| 3        | 4        |
+
+```python
+> results['data']
+
+[
+    (Decimal('1'), Decimal('2')),
+    (Decimal('3'), Decimal('4'))
+]
+
+```
+
+- `table`: An [Agate Table](https://agate.readthedocs.io/en/1.6.1/api/table.html) containing results. This is a powerful API for interacting with query results, but you may need to use the Agate docs to navigate it.
+
+### Examples
+
+All examples use this setup SQL:
+
+```sql
+{%- call statement('test_query', fetch_result=True) -%}
+
+    select 1 as my_col_1, 2 as my_col_2
+    union all
+    select 3 as my_col_1, 4 as my_col_2
+
+{%- endcall -%}
+
+{%- set test_query = load_result(name='test_query') -%}
+```
+
+We've included examples of how to use the [`log` macro](log) to print results to the command line in order to "see" what the object is.
+
+#### Returning the first row
+
+You can use both the `data` and the `table` properties to return this.
+
+Using the `data` property, both of these are equivalent:
```

Comment:
They're equivalent for [Jinja reasons](https://jinja.palletsprojects.com/en/2.11.x/templates/#variables), not because of anything special about `data` / `SQL Results`. (That's the implication I read here)